Crane/app/src/main/java/androidx/compose/samples/crane/calendar/Calendar.kt

```diff
@@ -70,14 +72,10 @@ fun Calendar(
     modifier: Modifier = Modifier
 ) {
     LazyColumn(modifier) {
-        item { Spacer(Modifier.height(32.dp)) }
         for (month in calendarYear) {
             itemsCalendarMonth(month = month, onDayClicked = onDayClicked)
-            item {
-                Spacer(Modifier.height(32.dp))
```

Comment:
Nit: I think the new logic is slightly different than the original, where the last set of items no longer has a `32dp` space below it (in addition to the navigation bars height)